src/presets/classic/factory.ts

```diff
@@ -28,6 +28,7 @@ export function createItem<S extends BSchemes>(
   }
   return <Item>{
     ...item,
-    subitems: factory.map((data, i) => createItem(data, i, context))
+    subitems: factory.map((data, i) => createItem(data, i, context)),
+    handler: Function()
```

Comment:
this is unsafe due to Content Security Policy directive: "script-src"

Reply:
also, can you squash commits into a single one with name of the format "fix: <your message>"? (unfortunately this part is not covered in the Contribution guide)